py-polars/polars/expr/datetime.py

```diff
@@ -1423,6 +1423,52 @@ def replace_time_zone(
             self._pyexpr.dt_replace_time_zone(time_zone, ambiguous._pyexpr)
         )
 
+    def convert_to_local_time_zone(
```

Comment:
not sure about this name - maybe just `to_naive_local`?

Reply:
I found this naive_local naming that I saw in chrono super confusing 😆

Reply:
True, but what does "local time zone" mean? A datetime can be local, not sure about this one

Maybe just `to_local_datetime`?

We'd also need this method to be added to docs/user-guide/transformations/time-series/timezones.md